Documentation/ceph-common-issues.md

```diff
@@ -814,6 +814,8 @@ If you still decide to configure an OSD on LVM, please keep the following in min
 
 - Disable `lvmetad.`
 - Avoid configuration of LVs from the host. In addition, don't touch the VGs and physical volumes that back these LVs. 
-- Avoid incrementing the `count` field of `storageClassDeviceSets` and create a new LV that backs a OSD simultaneously.
+- Avoid incrementing the `count` field of `storageClassDeviceSets` and create a new LV that backs an OSD simultaneously.
 
 You can know whether the above-mentioned tag exists tag with the command: `sudo lvs -o lv_name,lv_tags`. If the `lv_tag` field is empty in an LV corresponding to the OSD lv_tags, this OSD encountered the problem. In this case, please [retire this OSD](ceph-osd-mgmt.md#remove-an-osd) or replace with other new OSD before restarting.
+
+Rook no longer creates `ceph-volume` lvm mode OSD and creates OSD with `ceph-volume` raw mode instead. Although the lvm mode OSDs works continuously, using the raw mode OSDs is recommended because of the above-mentioned problem. You can replace the existing OSDs with raw mode OSDs by retiring them and adding new OSDs one by one. See the documents [Remove an OSD](ceph-osd-mgmt.md#remove-an-osd) and [Add an OSD on a PVC](ceph-osd-mgmt.md#add-an-osd-on-a-pvc).
```

Comment:
Existing lvm mode OSDs will still be supported during upgrade, right? The support being removed looks like it is for creating new OSDs. That support is only being removed from the osd prepare job. So it appears the existing OSDs will still be upgraded and are expected to be fully functional. Let's clarify that existing OSDs will continue to work after the upgrade.

Reply:
I reflected your comment.